details.html

```diff
@@ -179,17 +179,11 @@
                     alt="John Doe Avatar"
                   />
 
-                  <img
-                    src="./images/user3.png"
-                    class="avatar round small"
-                    alt="Shane Roe avatar"
-                  />
-
-                  <img
-                    src="./images/user4.png"
-                    class="avatar round small"
-                    alt="Bobby Green avatar"
-                  />
+                  <button
+                    class="avatar round small btn icon-only primary light more-users"
```

Comment:
ok, I see it's working, but calling a `button` `avatar` isn't quite right. This is not an avatar, it is a button. Find a way to make a button round without adding `avatar`  class.